packages/labs/analyzer/src/lib/javascript/jsdoc.ts

```diff
@@ -48,8 +49,20 @@ const parseNameDescRE = /^(?<name>^\S+)(?:\s?-\s+)?(?<description>[\s\S]*)$/m;
 // Regex for parsing optional name and description from JSDoc comments, where
 // the dash is required before the description (syntax for `@slot` tag, whose
 // default slot has no name)
-const parseNameDashDescRE =
-  /^(?<name>^\S*)?(?:\s+-\s+(?<description>[\s\S]*))?$/m;
+const parseNameDashDescRE = /^(?<name>^\S*)?(?:\s+-\s+(?<description>.+))?$/m;
+
+// Regex for parsing optional name, default, and description from JSDoc comments
+const parseNameDefaultDescRE =
+  /^\[(?<name>[^[\s=]+)=?(?<defaultValue>[^\]]+)?\](?:\s+-\s+(?<description>.+))?/m;
```

Comment:
don't you actually want the subgroup to be optional rather than the individual `=` to be optional?

i.e. instead of `=?(?<defaultValue>whatever)?`, you want `(?:=(?<defaultValue>whatever))?`

Reply:
giv'er another look, if you please